qa/python_models/init_args/model.py

```diff
@@ -25,36 +25,34 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 import os
+
 import numpy as np
 import triton_python_backend_utils as pb_utils
 
 
 def check_init_args(args):
     expected_args = {
-        'model_name':
-            'init_args',
-        'model_instance_name':
-            'init_args_0',
-        'model_instance_kind':
-            'CPU',
-        'model_instance_device_id':
-            '0',
-        'model_repository':
-            os.getenv("TRITON_DIR", "/opt/tironserver") +
-            '/qa/L0_backend_python/models/init_args',
-        'model_version':
-            '1'
+        "model_name": "init_args",
+        "model_instance_name": "init_args_0",
+        "model_instance_kind": "CPU",
+        "model_instance_device_id": "0",
+        "model_repository": os.getenv("TRITON_DIR", "/opt/tironserver")
```

Comment:
probably unused since the test presumably passes, but might as well fix it:
```suggestion
        "model_repository": os.getenv("TRITON_DIR", "/opt/tritonserver")
```